tests/spec/core/figures-spec.js

```diff
@@ -22,6 +22,42 @@ describe("Core - Figures", function() {
     expect(captions.item(0).textContent).toEqual("Figure 1 PREFIG");
     expect(captions.item(1).textContent).toEqual("Figure 2 IMGTIT");
   });
+
+  it("creates autolinks from the anchor to the figure", async () => {
+    const ops = {
+      config: makeBasicConfig(),
+      body:
+      makeDefaultBody() +
```

Comment:
Nit: use multiline template strings, instead of concatenation with "+" ... makes code more easy to change later.

Reply:
ok